site/_layouts/documentation.html

```diff
@@ -331,7 +331,7 @@ <h3>Extending Bazel</h3>
                 <ul class="collapse sidebar-nav sidebar-submenu" id="starlark-practices">
                   <li><a href="/versions/{{ current_version }}/skylark/bzl-style.html">.bzl style guide</a></li>
                   <li><a href="/versions/{{ current_version }}/skylark/testing.html">Testing extensions</a></li>
-                  <li><a href="https://github.com/bazelbuild/stardoc" target="_blank">Documenting rules with Stardoc</a></li>
+                  <li><a href="https://github.com/bazelbuild/stardoc">Documenting rules with Stardoc</a></li>
```

Comment:
Did you want rel=noopener here too?

Reply:
No, I removed the `target="_blank"` here, since none of the other links on the page uses it too.